### PR TITLE
Add routines and index for platform manipulation

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Depends-On: https://github.com/thoth-station/storages/pull/1800

## This introduces a breaking change

- [ ] Yes
- [x] No

## This should yield a new module release

- [x] Yes
- [ ] No

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -2514,6 +2514,23 @@ def get_python_package_version_dependents_all(
 
             return result
 
+    def python_package_version_depends_on_platform_exists(self, platform: str) -> bool:
+        """Check if the given platform has some records in the database."""
+        with self._session_scope() as session:
+            return session.query(exists().where(DependsOn.platform == platform)).scalar()
+
+    def get_python_package_version_platform_all(self) -> List[str]:
```
Comment (fridex) [inline thoth/storages/graph/postgres.py:2522]: Hope these two methods conform to the naming convention.

Comment (pacospace) [reply thoth/storages/graph/postgres.py:2522]: Yes they do :1st_place_medal:

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
recheck

**Pull request merged**